deploy/charts/external-secrets/README.md

```diff
@@ -59,3 +59,4 @@ The command removes all the Kubernetes components associated with the chart and
 | serviceAccount.create | bool | `true` | Specifies whether a service account should be created. |
 | serviceAccount.name | string | `""` | The name of the service account to use. If not set and create is true, a name is generated using the fullname template. |
 | tolerations | list | `[]` |  |
+| priorityClassName | string | `""` | Pod priority class name. |
```

Comment:
This table is in alphabetical order, let's move this to be right after podSecurityContext.